test/integration/targets/collections/runme.sh

```diff
@@ -13,22 +13,22 @@ export INVENTORY_PATH="$ipath"
 
 echo "--- validating callbacks"
 # validate FQ callbacks in ansible-playbook
-ANSIBLE_CALLBACK_WHITELIST=testns.testcoll.usercallback ansible-playbook noop.yml | grep "usercallback says ok"
+ANSIBLE_CALLBACK_ENABLED=testns.testcoll.usercallback ansible-playbook noop.yml | grep "usercallback says ok"
```

Comment:
I'm worried that this change turns plural name into a singular form. Can this be avoided?

Reply:
yes, see answer above